### PR TITLE
layers

### PR DESCRIPTION
**Pull request by segunjkf**

### CODE REVIEW

File: level2/variable.tf [hunk 1]
```diff
@@ -0,0 +1,9 @@
+variable "env_code" {}
+
+variable "instance_type" {}
+
+variable "ami" {}
+
+variable "sg_cidr" {}
+
+variable "image_name" {}
```
Comment (geoffrey1330) [inline level2/variable.tf]: Add newline at the end of file

File: level1/outputs.tf [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    value = aws_subnet.public[*].id
+}
+
+output "subnet_id_private" {
+    value = aws_subnet.private[*].id
+}
+
+output "vpc_cidr" {
+    value = var.vpc_cidr
+}
```
Comment (geoffrey1330) [inline level1/outputs.tf]: Add newline at the end of file

File: level1/terraform.tfvars [hunk 3]
```diff
@@ -5,11 +5,3 @@ vpc_cidr = "10.0.0.0/16"
...
 
 public_cidr = ["10.0.1.0/24", "10.0.2.0/24"]
-
-instance_type = "t3.micro"
-
-ami = "ami-005e54dee72cc1d00"
-
-sg_cidr = "102.89.45.199/32"
-
-image_name = "amzn2-ami-hvm-*-x86_64-gp2"
```
Comment (geoffrey1330) [inline level1/terraform.tfvars:15]: Add newline at the end of file

File: level1/variable.tf [hunk 4]
```diff
@@ -6,10 +6,3 @@ variable "private_cidr" {}
 
 variable "public_cidr" {}
 
-variable "instance_type" {}
-
-variable "ami" {}
-
-variable "sg_cidr" {}
-
-variable "image_name" {}
```
Comment (geoffrey1330) [inline level1/variable.tf:15]: Add newline at the end of file

File: level2/data.tf [hunk 5]
```diff
@@ -0,0 +1,9 @@
+data "terraform_remote_state" "layer1" {
+  backend = "s3"
+
+  config = {
+    bucket = "cloud-learning-bucket112"
+    key    = "layer1.tfstate"
+    region = "eu-west-1"
+  }
+}
```
Comment (geoffrey1330) [inline level2/data.tf]: Add newline at the end of file

File: level2/provider.tf [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+    region         = "eu-west-1"
+    dynamodb_table = "terraform-lock"
+  }
+
+  required_version = ">= 1.1.9"
+}
+
+provider "aws" {
+  region = "eu-west-1"
+}
```
Comment (geoffrey1330) [inline level2/provider.tf]: Add newline at the end of file

Comment (segunjkf) [reply level2/provider.tf]: > Add newline at the end of file

check again

**Review by geoffrey1330 — APPROVED** [review]

**Pull request merged**